IdentityServer/Quickstart/Account/ExternalUserInfo.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IdentityServer.Quickstart.Account
+{
+    /// <summary>
+    /// The viewmodel for /connect/userinfo
+    /// </summary>
+    public class ExternalUserInfo
+    {
+        /// <summary>
+        /// Gets or sets the sub.
+        /// </summary>
+        /// <value>
+        /// The sub.
+        /// </value>
+        public string sub { get; set; }
```

Comment:
Please use [JsonProperty("sub")] annotation and use proper naming convention. Same goes for the others